test/integration/bun-types/fixture/spawn.ts

```diff
@@ -179,16 +179,11 @@ function depromise<T>(_promise: Promise<T>): T {
   tsd.expectType<number>(proc.stdin);
 }
 tsd.expectAssignable<PipedSubprocess>(Bun.spawn([], { stdio: ["pipe", "pipe", "pipe"] }));
-tsd.expectNotAssignable<PipedSubprocess>(Bun.spawn([], { stdio: ["inherit", "inherit", "inherit"] }));
```

Comment:
Rather than removing these, they could be switched to `expectAssignable` with `// @ts-expect-error`?

Reply:
We could, but in this case they weren't super relevant since the tests for ReadableSubprocess, WritableSubprocess and NullSubprocess existed just below.

These checks were already not actually doing anything, so it was ultimately just noise